src/main/java/ca/corefacility/bioinformatics/irida/model/enums/GroupByFormat.java

```diff
@@ -0,0 +1,44 @@
+package ca.corefacility.bioinformatics.irida.model.enums;
+
+/**
+ * Defines a set of formats which admin statistics
+ * can be grouped by
+ *
+ */
+
+public enum GroupByFormat {
```

Comment:
Would it make sense to put all this stuff into `StatisticTimePeriod`?  You already have these exact values there so it's a bit of a duplication.

If you did that you could just add the `groupByFormat` property to that enum.

Reply:
Yeah that's a good idea. :) I will try it out and see how it goes. Thanks

Reply:
Updated in c5a00bcd1e78198ca591eaa499e2e494dd1cd5ca